data/changelog/odoc/2023-10-02-odoc-2.3.1.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Odoc 2.3.1
+title: `odoc` 2.3.1
```

Comment:
@christinerose Isn't this reserved for the CLI tool, and Odoc is the name of the projects?

Reply:
According to Jon, `odoc` is the proper name of the tool, unless he's changed his mind.